brainstormforce/ultimate-addons-for-gutenberg
# Feat: Added device icons on editor to manage responsive screens and Highlight breadcrumbs

### Description
Improvement: Add responsive icons and highlight breadcrumbs on editor

### Test Cases -

- On editor set wordpress preview to any device
- Check our device icons are activated accordingly or not.

### Test Cases -

- Change device and check screens are properly updating or not

### Test Cases -

- Drag and drop any Spectra block
- Check responsive options
- Change devices and check are they properly working with our feature

### Types of changes
non-breaking change which adds functionality

### How has this been tested?


### Checklist:
